tests-functional/clients/status_backend.py

```diff
@@ -190,8 +190,11 @@ def init_status_backend(self):
             "logEnabled": True,
             "logLevel": "DEBUG",
             "apiLogging": True,
-            "wakuFleetsConfigFilePath": option.waku_fleets_config,
         }
+
+        if option.waku_fleets_config and option.waku_fleet:
+            data["wakuFleetsConfigFilePath"] = option.waku_fleets_config
```

Comment:
We can just always pass the option value. When it's empty, `status-go` will use the hard-coded fleets.
```suggestion
        data["wakuFleetsConfigFilePath"] = option.waku_fleets_config
```